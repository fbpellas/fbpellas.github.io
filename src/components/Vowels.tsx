import * as React from 'react';
import { IPA_DEFINITION } from '../constants';
import { vowels } from '../data/phonemes';
import { PhonemesTable } from './PhonemesTable';
import { Tooltip } from './Tooltip';

export interface VowelsProps {
  setPageAndClear: (hash: string) => void;
}

const Vowels: React.FC<VowelsProps> = (props) => {
  const { setPageAndClear } = props;

  return (
    <div className="block-2">
      <div className="article">
        <h2 className="h3-title vowels-title">Vowels</h2>
        <br />
        <div>
          Vowels are a set of unblocked sounds. They consist of the letters A, E, I, O, U (sometimes Y). The{' '}
          <Tooltip definition={IPA_DEFINITION} word="IPA" /> lists 20 phonemes categorized as long, short, and{' '}
          <a className="clickable-page" href="#diphthongs" onClick={() => setPageAndClear('diphthongs')}>
            diphthongs
          </a>
          .
        </div>
        <br />
        <div>
          Vowels can sometimes be categorized as lax (short) and tense (long) depending on how much effort the lips and
          tongue make when producing the sound.
        </div>
        <br />
        <div>
          Listed below are the phonemes that are widely used in the American English language. Some words might vary in
          phonemes depending on regional dialects.
        </div>
        <PhonemesTable data={vowels} />
      </div>
    </div>
  );
};

export { Vowels };
