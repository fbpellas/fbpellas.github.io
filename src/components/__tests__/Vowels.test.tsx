import React from 'react';
import { render } from '@testing-library/react';
import { Vowels } from '../Vowels';

test('Vowels', () => {
  const fn = jest.fn();
  const { container } = render(<Vowels setPageAndClear={fn} />);
  expect(container).toMatchSnapshot();
});
