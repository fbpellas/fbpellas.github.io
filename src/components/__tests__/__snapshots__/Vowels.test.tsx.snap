// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vowels 1`] = `
<div>
  <div
    class="block-2"
  >
    <div
      class="article"
    >
      <h2
        class="h3-title vowels-title"
      >
        Vowels
      </h2>
      <br />
      <div>
        Vowels are a set of unblocked sounds. They consist of the letters A, E, I, O, U (sometimes Y). The
         
        <div
          class="text-inline bottom-dot"
        >
          IPA
        </div>
         lists 20 phonemes categorized as long, short, and
         
        <a
          class="clickable-page"
          href="#diphthongs"
        >
          diphthongs
        </a>
        .
      </div>
      <br />
      <div>
        Vowels can sometimes be categorized as lax (short) and tense (long) depending on how much effort the lips and tongue make when producing the sound.
      </div>
      <br />
      <div>
        Listed below are the phonemes that are widely used in the American English language. Some words might vary in phonemes depending on regional dialects.
      </div>
      <table>
        <thead>
          <tr>
            <th>
              <div
                class="text-center"
              >
                Phonemes
              </div>
            </th>
            <th
              class="text-center"
            >
              <div
                class="text-center"
              >
                Grapheme
              </div>
            </th>
            <th
              class="text-center"
            >
              Examples
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /iː/
            </td>
            <td>
              e, ee, ea, ey, ie, y
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  gr
                  <u>
                    ee
                  </u>
                  n
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  sh
                  <u>
                    ee
                  </u>
                  p
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  t
                  <u>
                    ea
                  </u>
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  k
                  <u>
                    ey
                  </u>
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                funn
                <u>
                  y
                </u>
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /ɪ/
            </td>
            <td>
              i, y, ui, u
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  p
                  <u>
                    i
                  </u>
                  nk
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  s
                  <u>
                    y
                  </u>
                  mbol
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  b
                  <u>
                    ui
                  </u>
                  ld
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  b
                  <u>
                    u
                  </u>
                  sy
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                q
                <u>
                  ui
                </u>
                lt
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /ʊ/
            </td>
            <td>
              oo, ou, u
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  w
                  <u>
                    oo
                  </u>
                  d
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  c
                  <u>
                    ou
                  </u>
                  ld
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  f
                  <u>
                    ul
                  </u>
                  l
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  b
                  <u>
                    oo
                  </u>
                  k
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                p
                <u>
                  u
                </u>
                sh
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /uː/
            </td>
            <td>
              u, o, oo, ou
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  bl
                  <u>
                    u
                  </u>
                  e
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  wh
                  <u>
                    o
                  </u>
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  n
                  <u>
                    oo
                  </u>
                  n
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  y
                  <u>
                    ou
                  </u>
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                gr
                <u>
                  ou
                </u>
                p
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /e/
            </td>
            <td>
              e, ea, ai
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  m
                  <u>
                    e
                  </u>
                  n
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  br
                  <u>
                    ea
                  </u>
                  d
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  m
                  <u>
                    e
                  </u>
                  t
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  s
                  <u>
                    ai
                  </u>
                  d
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                h
                <u>
                  ea
                </u>
                d
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /ə/
            </td>
            <td>
              (schwa sound)
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  b
                  <u>
                    a
                  </u>
                  nana
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  th
                  <u>
                    e,
                  </u>
                   ev
                  <u>
                    e
                  </u>
                  r
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  p
                  <u>
                    o
                  </u>
                  lice
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                <u>
                  u
                </u>
                pon
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /ɜː/
            </td>
            <td>
              i(r), u(r), ea(r)
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  b
                  <u>
                    ir
                  </u>
                  d
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  h
                  <u>
                    ur
                  </u>
                  t
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  l
                  <u>
                    ear
                  </u>
                  n
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  <u>
                    ear
                  </u>
                  th
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                f
                <u>
                  ur
                </u>
                niture
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /ɔː/
            </td>
            <td>
              o(r), oo(r), o(r)e, oa(r)
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  m
                  <u>
                    or
                  </u>
                  ning
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  fl
                  <u>
                    oor
                  </u>
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  w
                  <u>
                    ore
                  </u>
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  b
                  <u>
                    oar
                  </u>
                  d
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                sp
                <u>
                  or
                </u>
                t
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /æ/
            </td>
            <td>
              a, au, ai
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  <u>
                    a
                  </u>
                  pple
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  b
                  <u>
                    a
                  </u>
                  ck
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  pl
                  <u>
                    ai
                  </u>
                  d
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  l
                  <u>
                    au
                  </u>
                  gh
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                <u>
                  a
                </u>
                fter
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /ʌ/
            </td>
            <td>
              u, o, oo, oe
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  <u>
                    u
                  </u>
                  nder
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  h
                  <u>
                    o
                  </u>
                  ney
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  fl
                  <u>
                    oo
                  </u>
                  d
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  d
                  <u>
                    oe
                  </u>
                  s
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                f
                <u>
                  u
                </u>
                n
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /aː/
            </td>
            <td>
              a(r), o, ea(r), ow
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  h
                  <u>
                    ar
                  </u>
                  d
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  sh
                  <u>
                    ar
                  </u>
                  k
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  h
                  <u>
                    ear
                  </u>
                  t
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  kn
                  <u>
                    ow
                  </u>
                  ledge
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                b
                <u>
                  o
                </u>
                dy
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              /ɒ/
            </td>
            <td>
              o
            </td>
            <td>
              <svg
                class="play-icon"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M10 18a8 8 0 100-16 8 8 0 000 16zM9.555 7.168A1 1 0 008 8v4a1 1 0 001.555.832l3-2a1 1 0 000-1.664l-3-2z"
                  fill-rule="evenodd"
                />
              </svg>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  l
                  <u>
                    o
                  </u>
                  t
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  cl
                  <u>
                    o
                  </u>
                  th
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  h
                  <u>
                    o
                  </u>
                  nest
                </div>
                ,
                 
              </div>
              <div
                class="text-inline"
              >
                <div
                  class="text-inline nowrap"
                >
                  rob
                  <u>
                    o
                  </u>
                  t
                </div>
                ,
                 
              </div>
              <div
                class="text-inline nowrap"
              >
                st
                <u>
                  o
                </u>
                p
              </div>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
